test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -640,7 +641,7 @@ public void TestTreeEnsembleCombiner()
             CombineAndTestTreeEnsembles(dataView, fastTrees);
         }
 
-        [Fact]
+        [NotArm32Fact("Not supported on Arm32")]
```

Comment:
This should have the same comment above - about RyuJit with the link to the issue.

Reply:
Good point. Fixed it.